kimchi/src/verifier_index.rs

```diff
@@ -238,7 +238,7 @@ where
             endo: self.cs.endo,
             lookup_index,
             linearization: self.linearization.clone(),
-            fr_sponge_params: self.cs.fr_sponge_params.clone(),
+            fr_sponge_params: self.srs.scalar_sponge_params.clone(),
             fq_sponge_params: self.fq_sponge_params.clone(),
```

Comment:
I guess `fq_sponge_params` could follow the same treatment right? The SRS could contain both parameters